.github/workflows/sql-jdbc-test-and-build-workflow.yml

```diff
@@ -39,7 +32,7 @@ jobs:
         cp ./build/libs/*.jar sql-jdbc-builds
 
     - name: Upload Artifacts
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v3
       with:
-        name: sql-jdbc
+        name: sql-jdbc-java-${{ matrix.java }}
```

Comment:
what is ${{ matrix.java }} produce?

Reply:
11 or 17
You can see results here: https://github.com/opensearch-project/sql-jdbc/actions/runs/5203880109